llvm/lib/IR/Attributes.cpp

```diff
@@ -473,6 +473,14 @@ std::optional<unsigned> Attribute::getVScaleRangeMax() const {
   return unpackVScaleRangeArgs(pImpl->getValueAsInt()).second;
 }
 
+unsigned Attribute::getVScaleValue() const {
+  std::optional<unsigned> VScale = getVScaleRangeMax();
+  if (VScale && *VScale == getVScaleRangeMin())
+    return *VScale;
+
+  return 0;
+}
```

Comment:
I think this API would be more useful if you put it on Function instead? Then you don't have to do the getFnAttribute + isValid dance multiple times.

Reply:
Done.